report/src/discussion/discussion.tex

```diff
@@ -1,37 +1,40 @@
 \chapter{Discussion}\label{chap:discussion}
 This project involved two main phases: initially, we designed LoFi and HiFi prototypes for a tablet-based meeting room management system, which were well-received by the stakeholder, Mariendal IT.
-Through the prototype evaluation described in Section \ref{sec:prototype_evaluation}, it became evident that automating presence detection was crucial to fully address their needs.
-This led to the problem formulation in Section \ref{sec:problem_statement}.
+By choosing a user-centered prototype evaluation, described in Section \ref{sec:prototype_evaluation}, we examined the stakeholder's problems from their point of view.
+One benefit of using a user-centered prototype evaluation is that the users can provide insights into the inner workings of the application domain that would otherwise remain undiscovered.
```

Comment:
Yes! But shouldn't this have been written earlier in the report?

Reply:
Maybe, perhaps. I do not think it would benefit the report mentioning it earlier. Besides, at that point in the process we did not know it would be problematic. I think that the discussion is the perfect place to discuss what we might have gained by using that specific method. We already have academic reasons for using the method, but this clarifies the benefits for this project.

Reply:
OK. Fine reason. Thanks.